Prog/Block00_bpk.sas

```diff
@@ -0,0 +1,100 @@
+/************************************************************************
+  Program:  Block00_bpk.sas
+  Library:  General
+  Project:  DC Data Warehouse
+  Author:   J. Dev
+  Created:  02/16/17
+  Version:  SAS 9.2
+  Environment:  Windows
+  
+  Description:  Create Census 2000 block to Bridge Park area
+  correspondence file.
+
+  Adds correspondence format $bk0bpk. to local General library.
+
+  Modifications:
+************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( OCTO )
+
+*options obs=50;
+
+data General.Block00_bpk 
+  (label="Census 2000 blocks (GeoBlk2000) to Bridge Park area (bridgepk) correspondence file");
+
+  set OCTO.Block00_bpk;
```

Comment:
The instructions say to save the layer created by joining the geography to 2000 census block points in the OCTO\Maps directory on L:\. That data set is created using Stat/Transfer of the corresponding .dbf (and is saved on my D:\ drive). The corresponding file joined to 2010 census block points is in the General\Maps\Census 2010 directory on L:\. 

Should the layers and data files be saved in General\Maps?